pineappl_py/docs/source/recipes.rst

```diff
@@ -13,7 +13,7 @@ How can I convolve a given PineAPPL grid with my PDF?
     import lhapdf
     g = pineappl.grid.Grid.read("path/to/grid.pineappl.lz4")
     pdf = lhapdf.mkPDF("YourPDF", 0)
-    bins = g.convolve(pdf.xfxQ2, pdf.xfxQ2, pdf.alphasQ2)
+    bins = g.convolve_with_one(2212, pdf.xfxQ2, pdf.alphasQ2)
```

Comment:
We should make these snippets running at some point (not now) - e.g. turning the page into a notebook - to see if our examples actually work (which they were not before - I hope they do now :see_no_evil: ). Note the dependency on lhapdf, but as I found out out the other day in yadism - running the notebook locally should be sufficient for RTD

Reply:
There are doctests:
https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html